conmon-rs/server/src/console_or_iostreams.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{console::Console, iostreams::IOStreams};
+use crossbeam_channel::Sender;
+use std::sync::mpsc::Receiver;
+
+pub enum ConsoleOrIOStreams {
+    Console(Console),
+    IOStreams(IOStreams),
+}
```

Comment:
I almost think these should be

TerminalIO
NonTerminalIO

WDYT @saschagrunert @rphillips

Reply:
fair enough. I'm having trouble figuring out an alternative (TerminalIO and SreamIO?) so we can just leave it for now

Reply:
I remember when I started working on conmon, the use of the word `console` was confusing. It was always paired with --tty or --terminal in runc, but never referenced terminals

Reply:
Changed it to:
```rust
pub enum ContainerIO {
    Terminal(Console),
    Streams(Streams),
}
```